front/src/components/AdminPanel/Panel/CompetitionDetails/CompetitionDetails.js

```diff
@@ -14,21 +14,27 @@ function CompetitionDetails(props) {
         <div>
             <p className="editLinkText">
                 <span className="panel-detail-header">Szczegóły zawodów</span>
-                <span><a
-                        href={"/admin/panel/" + _id + "/details/edit"}>
-                        edytuj
-                     </a>
+                <span>
+                    <a href={"/admin/panel/" + _id + "/details/edit"}>edytuj</a>
```

Comment:
These are purely whitespace changes, do you have Prettier up and running? I don't see the reason why it wouldn't format these lines.

Reply:
I think this component was written by @AleksandraRolka , I don't remember making any changes there - in that case I'm nit sure whether there is something wrong with mine or her configurations. Another thing is that this line should probably be changed anyway as we cannot use <a> tags and links to navigate, as it refreshes the whole page - but I havent noticed that before, as we haven't been using redux at the beginning